frontend/src/components/StaticMap/StaticMap.js

```diff
@@ -29,7 +29,7 @@ const StaticMap = ({ geoJSON, square, height: mapHeight, areaId }) => {
 
   useEffect(() => {
     const handleResize = () => setSize(generateSize(ref, square, mapHeight))
-    handleResize()
+    setTimeout(() => handleResize(), 0)
```

Comment:
This fixes the "blank on first visit" issue on Firefox and Chrome, but it's still black most of the time on Safari. On Safari, `ref` is still null when the timeout fire. Raising the timeout delay to 1000ms makes it render correctly on Safari, but that's a fragile solution. I think the right solution is either to [use a callback ref](https://reactjs.org/docs/hooks-faq.html#how-can-i-measure-a-dom-node) in combination with the current event listener or to copy one of the many useSize/useDimension/useResize hooks out there.